cfalertdialog/src/main/java/com/crowdfireapp/cfalertdialog/CFAlertDialog.java

```diff
@@ -833,6 +839,9 @@ private float getCornerRadius(){
                 break;
         }
 
+        // Use the corner radius from params if provided.
+        if (params.dialogCornerRadius != 1) cornerRadius = params.dialogCornerRadius;
```

Comment:
What is this value `1` ? If this is the default corner radius, set it to a dimens constant